rotkehlchen/chain/avalanche/manager.py

```diff
@@ -135,11 +140,19 @@ def get_transaction_receipt(
                 receipt_log['logIndex'] = receipt_log.pop('log_offset', None)
                 receipt_log['transactionIndex'] = 0
                 tx_receipt['log_events'][index] = receipt_log
-        except (DeserializationError, ValueError) as e:
-            raise RemoteError(
-                f'Couldnt deserialize transaction receipt '
-                f'data from covalent {tx_receipt} due to {str(e)}',
-            ) from e
+        except (DeserializationError, ValueError, KeyError) as e:
+            msg = str(e)
+            if isinstance(e, KeyError):
+                msg = (
+                    f'Couldn\'t find required data in the covalent\'s '
+                    f'response {tx_receipt}. Missing key: {msg}'
+                )
+            else:
+                msg = (
+                    f'Couldnt deserialize transaction receipt data from '
+                    f'covalent {tx_receipt} due to {msg}'
+                )
+            raise RemoteError(msg) from e
```

Comment:
Hey @nebolax come on man please look at what I am writing in the reviews. It's super super simple. I pointed you to a specific example and all you had to do was copy it. Instead you are doing other stuff and putting more assignments of msg than needed, adding extra logic and the typo I pointed to is still not fixed. You were writting **the** Covalent which is wrong.

now please copy paste **exactly** what I am adding here and do nothing else.


```suggestion
        except (DeserializationError, ValueError, KeyError) as e:
            msg = str(e)
            if isinstance(e, KeyError):
                msg = f'missing key entry for {msg}'
                
             
            raise RemoteError(
                f'Couldnt deserialize transaction receipt data from '
                f'covalent {tx_receipt} due to {msg}'
            ) 
```